src/validations/rules/Writing_FedRAMP_Validations.md

```diff
@@ -0,0 +1,375 @@
+    </remarks>
+</responsible-party>
+```
+and the code block
+```xml
+<sch:rule
+    context="oscal:responsible-party"
+    doc:guide-reference="Guide to OSCAL-based FedRAMP System Security Plans §4.6-§4.11">
+    <sch:assert
+        diagnostics="responsible-party-has-role-diagnostic"
+        doc:guide-reference="Guide to OSCAL-based FedRAMP System Security Plans §4.6-§4.11"
+        id="responsible-party-has-role"
+        role="error"
+        test="exists(//oscal:role[@id eq current()/@role-id])">The role for a responsible party must exist.</sch:assert>
+    <sch:assert
+        diagnostics="responsible-party-has-party-uuid-diagnostic"
+        id="responsible-party-has-party-uuid"
+        role="error"
+        test="exists(oscal:party-uuid)">One or more parties must be identified for a responsibility.</sch:assert>
+    <sch:assert
+        diagnostics="responsible-party-has-definition-diagnostic"
+        id="responsible-party-has-definition"
+        role="error"
+        test="
+        every $p in oscal:party-uuid
+        satisfies exists(//oscal:party[@uuid eq $p])">Every responsible party must be defined.</sch:assert>
+    <sch:assert
+        diagnostics="responsible-party-is-person-diagnostic"
+        doc:guide-reference="Guide to OSCAL-based FedRAMP System Security Plans §4.6-§4.11"
+        id="responsible-party-is-person"
+        role="error"
+        test="
+        if (
+        current()/@role-id = (
+        'system-owner',
+        'authorizing-official',
+        'system-poc-management',
+        'system-poc-technical',
+        'system-poc-other',
+        'information-system-security-officer',
+        'authorizing-official-poc'
+        )
+        )
+        then
+        every $p in oscal:party-uuid
+        satisfies
+        exists(//oscal:party[@uuid eq $p and @type eq 'person'])
+        else
+        true()
+        ">For some roles responsible parties must be persons.</sch:assert>
+</sch:rule>
+```
+The `<rule>` sets the context for the subordinate assertions.
+
+The first (`id="responsible-party-has-role"`) assertion is an existential test for a `<role>` for the `@role-id`. The use of the `//` prefix in the path `//oscal:role[@id eq current()/@role-id]` means "any `<role>` element within the document" matching the context `@role-id`.
+
+The second (`id="responsible-party-has-party-uuid"`) assertion is just an existential test for `<party-uuid>`.
+
+The third (`id="responsible-party-has-definition"`) assertion is an existential test that every `<party-uuid>` reference (there may be more than one subordinate to `<responsible-party>`) references a `<party>` anywhere within the ndocument. 
+
+The fourth () assertion ensures that specific role types identify a responsible **person** (i.e., a `<party>` defined as a person).
+
+## Example 3 - remote resource reference
+
+This example uses remote HTTPS resource availability to ensure a CMVP citation is actually found on the related NIST web site.
```

Comment:
Spell out CMVP (Cryptographic Module Validation Program).. When I first read the sentence I had no idea what you were talking about.

Reply:
Acronym spelled out.